starfish/core/spots/FindSpots/blob.py

```diff
@@ -84,7 +85,7 @@ def __init__(
         except ValueError:
             raise ValueError("Detector method must be one of {blob_log, blob_dog, blob_doh}")
 
-    def image_to_spots(self, data_image: Union[np.ndarray, xr.DataArray]) -> SpotAttributes:
+    def image_to_spots(self, data_image: Union[np.ndarray, xr.DataArray]) -> PerImageSpotResults:
```

Comment:
PerTileSpotResults?

Reply:
Would we call it tile though since we don't include a z value? I went back a forth about this but all your TileKey TileFetcher code includes a r/ch/z for each tile so I wasn't sure it was the same

Reply:
PerImageSliceSpotResults?

fwiw, I thought we should call n-dimensional tensors images (plural) rather than singular, which I think would have avoided this wonkiness.  You should consult with the proponents of calling the entire n-dimensional tensor "image". :)

If they say nothing, then I would go with your judgement.